### PR TITLE
[FEAT] Add a SeqAn Version Badge to every inhouse app.

### PR DESCRIPTION
**Pull request by simonsasse**
Resolves partly: https://github.com/seqan/www.seqan.de/issues/67 

I highlighted seqan3 badge by reducing the opacity of the seqan2 badge by 30%.

### CODE REVIEW

**Comment by eseiler** [comment]
I think the badge could be a bit smaller (the box itself, font size seems good). I would also give seqan3 another color (green?).

Can you please rebase on master and also mark raptor?

**Review by eseiler — CHANGES_REQUESTED** [review]
see above

**Review by eseiler — APPROVED** [review]

File: assets/css/style.scss [hunk 1]
```diff
@@ -398,3 +398,17 @@ details.person[open] {
...
+// ====== App-page Style ======
+
+.seqan-version-badge-2, .seqan-version-badge-3{
+  color: black;
+  border-radius: 2px;
+  font-size: 80%;
+  padding: 2px;
+}
+.seqan-version-badge-2 {
+  background-color: $seqan-color;
```
Comment (marehr) [inline assets/css/style.scss:410]:
```suggestion
  background-color: rgb(253, 253, 150); // pastel yellow
  border: 1px solid color.scale(rgb(251, 251, 70), $lightness: -20%); // 20% darker color
```

File: assets/css/style.scss [hunk 2]
```diff
@@ -398,3 +398,17 @@ details.person[open] {
...
+}
+.seqan-version-badge-3 {
+  background-color: greenyellow;
```
Comment (marehr) [inline assets/css/style.scss:413]:
```suggestion
  background-color: #77dd77;  // pastel green
  border: 1px solid color.scale(#77dd77, $lightness: -20%); // 20% darker color
```

File: assets/css/style.scss [hunk 3]
```diff
@@ -398,3 +398,20 @@ details.person[open] {
...
+  padding: 0.33em;
+  box-shadow: black 0px 0px 1px inset;
+}
+.seqan-version-badge-2 {
+  background-color: rgb(#FDFD96); // pastel yellow
+  border: 1px solid color.scale(rgb(#FDFD96), $lightness: -20%); // 20% darker color
+}
+.seqan-version-badge-3 {
+  background-color: #77dd77;  // pastel green
+  border: 1px solid color.scale(#77dd77, $lightness: -20%); // 20% darker color
```
Comment (marehr) [inline assets/css/style.scss:416]:
```suggestion
  border: 1px solid darken(#77dd77, 20%); // 20% darker color
```

File: assets/css/style.scss [hunk 4]
```diff
@@ -398,3 +398,20 @@ details.person[open] {
...
+
+.seqan-version-badge-2, .seqan-version-badge-3{
+  color: black;
+  border-radius: 3px;
+  font-size: 80%;
+  padding: 0.33em;
+  box-shadow: black 0px 0px 1px inset;
+}
+.seqan-version-badge-2 {
+  background-color: rgb(#FDFD96); // pastel yellow
```
Comment (marehr) [inline assets/css/style.scss:411]:
```suggestion
  background-color: #FDFD96; // pastel yellow
```

File: assets/css/style.scss [hunk 5]
```diff
@@ -398,3 +398,20 @@ details.person[open] {
...
+  padding: 0.33em;
+  box-shadow: black 0px 0px 1px inset;
+}
+.seqan-version-badge-2 {
+  background-color: #FDFD96; // pastel yellow
+  border: 1px solid darken(#FDFD96, 20%); // 20% darker color
+}
+.seqan-version-badge-3 {
+  background-color: #77dd77;  // pastel green
+  border: 1px solid darken(#77dd77, 20%); // 20% darker color
```
Comment (marehr) [inline assets/css/style.scss:416]:
```suggestion
  background-color: #C1E1C1;  // pastel green
  border: 1px solid darken(#C1E1C1, 20%); // 20% darker color
```

**Comment by marehr** [comment]
@eseiler , @simonsasse 

We have now three versions:

* https://www-seqan-de-87j4jjqej-seqan.vercel.app/apps/ (original) 
* https://www-seqan-de-7u92dud6m-seqan.vercel.app/apps/ (pastel yellow & green)
* https://www-seqan-de-95utwrzu1-seqan.vercel.app/apps/ (same pastel yellow but a more pastel green)

Which version do you prefer?

**Comment by eseiler** [comment]
I think

> * https://www-seqan-de-95utwrzu1-seqan.vercel.app/apps/ (same pastel yellow but a more pastel green)

**Comment by simonsasse** [comment]
I Personally like this one https://www-seqan-de-7u92dud6m-seqan.vercel.app/apps/ because of the more obvious green.
but I thats not a strong opinion :)

**Comment by simonsasse** [comment]
> I think
> 
> > * https://www-seqan-de-95utwrzu1-seqan.vercel.app/apps/ (same pastel yellow but a more pastel green)

Lets go with this :)

**Comment by eseiler** [comment]
I forgot if we decided something today, but I'd say you decide :)
Can you please rebase (again) and add the seqan3 badge to igenvar?

**Comment by eseiler** [comment]
Let's do it like this, we can always change it later :)

**Pull request merged**